corehq/apps/enterprise/interface.py

```diff
@@ -0,0 +1,205 @@
+
+from django.utils.translation import ugettext_noop as _
+from django.db.models.aggregates import Count
+from corehq.apps.reports.generic import GenericTabularReport
+from corehq.apps.enterprise.dispatcher import EnterpriseReportDispatcher
+from corehq.apps.reports.datatables import DataTablesColumn, DataTablesHeader
+from corehq.apps.accounting.filters import DateCreatedFilter, NameFilter
+from corehq.apps.sms.models import INCOMING, OUTGOING
+
+
+from corehq.apps.smsbillables.filters import (
+    DateSentFilter,
+    GatewayTypeFilter,
+    HasGatewayFeeFilter,
+    ShowBillablesFilter,
+)
+from corehq.apps.enterprise.filters import EnterpriseDomainFilter
+from corehq.apps.accounting.models import (
+    BillingAccount,
+    Subscription
+)
+from corehq.apps.smsbillables.models import (
+    SmsBillable,
+)
+
+from couchexport.models import Format
+from dimagi.utils.dates import DateSpan
+
+
+class EnterpriseSMSBillablesInterface(GenericTabularReport):
```

Comment:
Yeah. We decided to keep this as an interface because this is on par with the SMSBillablesInterface itself.

Reply:
Followed up with @biyeun , and she agreed that that distinction between a 'report' and an 'interface' is that a report is read-only, just presenting data, while an 'interface' is read/write and used to allow the user to modify the data it presents.  `SMSBillablesInterface` was misnamed with that understanding, and we should not repeat that mistake. 

While I understand wanting to tie the two together, the motivation for keeping `SMSBillablesInterface` separate from `EnterpriseSMSBillablesReport` is that they are distinct reports, and we don't want to risk exposing the admin-level data from `SMSBillablesInterface` on `EnterpriseSMSBillablesReport` -- hence there isn't much value to keeping their names consistent. If we need to echo any change from one to the other, it would violate what we're claiming above.